public/wp-tests-config.php

```diff
@@ -1,7 +1,23 @@
-<?php
+<?php // DO NOT EDIT THIS FILE! Please put customizations in the git-ignored wp-tests-config-local.php!
+
+// Allow local environment to override config.
+if ( file_exists( __DIR__ . '/wp-tests-config-local.php' ) ) {
+	require __DIR__ . '/wp-tests-config-local.php';
+} elseif ( file_exists( dirname( __DIR__ ) . '/wp-tests-config-local.php' ) ) {
+	require dirname( __DIR__ ) . '/wp-tests-config-local.php';
```

Comment:
I don't like using `__DIR__` here, at least for as long as core supports 5.2 (and as such people might want to test 5.2 compatibility). We could fix that separately in the future when the time is right.

Reply:
Reverted in aae1c81.